### PR TITLE
A radio input field which is generated by an input option list. Both,…

### PR DESCRIPTION
**Pull request by haogatyp**
… the value and the label can be transmitted by the form via a linked field.

If the value contains a URI starting with "http", a link "more information" will be displayed after the corresponding label text.

### CODE REVIEW

File: Classes/Domain/Model/DocumentFormField.php [hunk 1]
```diff
@@ -35,6 +35,8 @@ class DocumentFormField extends AbstractFormElement
 
     protected $dataType;
 
+    protected $linkedFieldUid;
```
Comment (albig) [inline Classes/Domain/Model/DocumentFormField.php]: Variable data type is missing.

File: Classes/Domain/Model/DocumentFormField.php [hunk 2]
```diff
@@ -184,4 +186,13 @@ public function setDataType($dataType)
     {
         $this->dataType = $dataType;
     }
+
```
Comment (albig) [inline Classes/Domain/Model/DocumentFormField.php]: @params and @return declaration is missing.

**Comment by haogatyp** [comment]
Due to changed requirements, the feature will be revised.

File: Classes/Domain/Model/DocumentFormField.php (hunk 1 above)
Comment (albig) [reply Classes/Domain/Model/DocumentFormField.php]: The type annotation above the declaration is used by extbase for validation of this object.
E.g.  
<pre>
 /**
  * @var string
  */
</pre>

The same for getter and setter.

**Pull request closed without merging**